addons/account/models/account.py

```diff
@@ -322,6 +332,12 @@ def onchange_internal_type(self):
         if self.internal_type == 'liquidity':
             self.reconcile = False
 
+    @api.onchange('internal_group')
+    def onchange_internal_type(self):
```

Comment:
You are erasing the method above having the same name. You could merge both together with `api.onchange('user_type_id')` as dependency, what do you think?